docs/strings.rst

```diff
@@ -135,6 +135,10 @@ mechanism.  The regular rules for writing apply; to get a fixed-width ASCII
 string, use ``numpy.string_``, and to get a variable-length ASCII string, use
 ``bytes``.
 
+Since Python 3 strings are encoded as UTF-8 by default. This causes some issues
```

Comment:
FYI, the python 3 str/python 2 unicode types have variable encodings (see https://www.python.org/dev/peps/pep-0393/), I would avoid mentioning how they are encoded.